src/Workspaces/Core/Portable/PublicAPI.Shipped.txt

```diff
@@ -139,7 +139,6 @@ abstract Microsoft.CodeAnalysis.Editing.SyntaxGenerator.TryCatchStatement(System
 abstract Microsoft.CodeAnalysis.Editing.SyntaxGenerator.TupleElementExpression(Microsoft.CodeAnalysis.SyntaxNode type, string name = null) -> Microsoft.CodeAnalysis.SyntaxNode
 abstract Microsoft.CodeAnalysis.Editing.SyntaxGenerator.TupleExpression(System.Collections.Generic.IEnumerable<Microsoft.CodeAnalysis.SyntaxNode> arguments) -> Microsoft.CodeAnalysis.SyntaxNode
 abstract Microsoft.CodeAnalysis.Editing.SyntaxGenerator.TypedConstantExpression(Microsoft.CodeAnalysis.TypedConstant value) -> Microsoft.CodeAnalysis.SyntaxNode
-abstract Microsoft.CodeAnalysis.Editing.SyntaxGenerator.TypeExpression(Microsoft.CodeAnalysis.ITypeSymbol typeSymbol) -> Microsoft.CodeAnalysis.SyntaxNode
```

Comment:
I might misremember things, but shouldn't we leave removed API in shipped.txt while adding a corresponding *REMOVED* entry in unshipped.txt until it ships?

Reply:
Oops. Yes.  I meant to copy, not cut here. Thanks!